cpp/src/parquet/bloom_filter_test.cc

```diff
@@ -242,6 +236,77 @@ TEST(OptimalValueTest, TestBloomFilter) {
       UINT32_C(1073741824));
 }
 
+int64_t HASHES_OF_LOOPING_BYTES_WITH_SEED_0[32] = {
```

Comment:
@pitrou these test is pasted from parquet-mr: https://github.com/apache/parquet-mr/commit/806037c080dc477798d157cd4a54a81240a85d37#diff-49a81f9fa74dcf80bbb13202cf9bf4c94ace2030920a2fce326b4ba879749c7e

Reply:
Well, we don't need to copy blindly what parquet-mr does :-)

Reply:
> Well, we don't need to copy blindly what parquet-mr does :-)

I think it's correct and standard, and it will be too long, just copying this piece of code is much trustworthy than hello world and different seed...